Tweaks/TweaksAssembly/BombWrapper.cs

```diff
@@ -234,43 +234,157 @@ public void Awake()
 						new TTKComponentSolver(component, bombModule, Tweaks.CurrentMode.Equals(Mode.Zen) ? Modes.initialTime : timerComponent.TimeRemaining);
 						break;
 
-					// Correct some mispositioned objects in older modules
+					// Correct some mispositioned objects in some regular modules
 					case "ForeignExchangeRates":
 					case "resistors":
 					case "CryptModule":
 					case "LEDEnc":
 						// This fixes the position of the module itself (but keeps the status light in its original location, which fixes it)
 						component.transform.Find("Model").transform.localPosition = new Vector3(0.004f, 0, 0);
 						break;
+					case "ColourFlash":
+					case "ColourFlashES":
+					case "ColourFlashPL":
+						// This fixes the position of the module itself (but keeps the status light in its original location)
+						component.transform.Find("ModuleBackground").transform.localPosition = Vector3.zero;
```

Comment:
I didn't realize this before but `Find()` returns a transform, so that `.transform` is redundant.
```suggestion
						component.transform.Find("ModuleBackground").localPosition = Vector3.zero;
```